subgraphs/aave-v2-forks/deploymentJsonContext.ts

```diff
@@ -9,11 +9,11 @@ function slugToTitle(slug: string): string {
 }
 
 export const deploymentJsonContext: DeploymentJsonContextInterface = {
```

Comment:
i think this file is supposed to be git ignored

Reply:
good catch, removing